src/components/carousel/CarouselPresenter.ts

```diff
@@ -1,9 +1,11 @@
+import React, {ReactNode} from 'react';
 import _ from 'lodash';
 import {CarouselProps, CarouselState} from './types';
 
-export function getChildrenLength(props: CarouselProps): number {
-  const length = _.get(props, 'children.length') || 0;
-  return length;
+type PropsWithChildren = CarouselProps & { children?: ReactNode }
```

Comment:
'children' are not already part of CarouselProps?

Reply:
[No](https://github.com/wix/react-native-ui-lib/blob/16afa97a1c6d511d398ee7c0b6a3195b9cca4d9b/src/components/carousel/types.tsx#L10) - It is fine in the component because the class uses a generic which looks like `readonly props: Readonly<P> & Readonly<{ children?: ReactNode }>;`